src/components/views/dialogs/CommunityPrototypeInviteDialog.tsx

```diff
@@ -171,44 +167,38 @@ export default class CommunityPrototypeInviteDialog extends React.PureComponent<
     public render() {
         const emailAddresses = [];
         this.state.emailTargets.forEach((address, i) => {
-            emailAddresses.push(
-                <Field
-                    key={i}
-                    value={address}
-                    onChange={(e) => this.onAddressChange(e, i)}
-                    label={_t("Email address")}
-                    placeholder={_t("Email address")}
-                    onBlur={() => this.onAddressBlur(i)}
-                />
-            );
+            emailAddresses.push(<Field
+                key={i}
+                value={address}
+                onChange={(e) => this.onAddressChange(e, i)}
+                label={_t("Email address")}
+                placeholder={_t("Email address")}
+                onBlur={() => this.onAddressBlur(i)}
+            />);
```

Comment:
What was it complaining about? This should be fine/valid.

Likewise below.

Reply:
missing trailing comma, thought this would be neater

Reply:
personally I'm not a fan of the mismatched open and close style. Can it be made happy with something like this?

```
thing.push((
    <Element
        prop={val}
    />
));
```